modules/store/package.json

```diff
@@ -19,8 +22,11 @@
   "homepage": "https://github.com/ngrx/platform#readme",
   "peerDependencies": {
     "@angular/core": "NG_VERSION",
+    "@angular-devkit/core": "NG_DEVKIT_VERSION",
```

Comment:
Remove these from the `package.json` as we are treating them as implicit dependencies instead of adding additional dependencies to `@ngrx/store`